openedx/core/djangoapps/programs/tests/test_backpopulate_program_credentials.py

```diff
@@ -0,0 +1,292 @@
+import json
+from unittest import skipUnless
+
+import ddt
+from django.conf import settings
+from django.core.management import call_command, CommandError
+from django.test import TestCase
+from edx_oauth2_provider.tests.factories import ClientFactory
+import httpretty
+import mock
+from provider.constants import CONFIDENTIAL
+
+from lms.djangoapps.certificates.api import MODES
+from lms.djangoapps.certificates.tests.factories import GeneratedCertificateFactory
+from openedx.core.djangoapps.programs.models import ProgramsApiConfig
+from openedx.core.djangoapps.programs.tests import factories
+from openedx.core.djangoapps.programs.tests.mixins import ProgramsApiConfigMixin
+from student.tests.factories import UserFactory
+
+
+COMMAND_MODULE = 'openedx.core.djangoapps.programs.management.commands.backpopulate_program_credentials'
```

Comment:
![the apollo command module](http://postcardvalet.com/wp-content/photoblog/arlomidgett-apollo14-command-1600.jpg)